cmd/tendermint/commands/run_node.go

```diff
@@ -35,27 +35,27 @@ var (
 func init() {
 
 	// configuration options
-	runNodeCmd.Flags().StringVar(&moniker, "moniker", config.GetString("moniker"),
```

Comment:
Oops. Yeah I get why this a problem now. The only option I see where the defaults can be initialized from config is if we define a persistent config file location (which could be modified with a tendermint command). This way the config file would be initialized before the flags and could be read in... maybe not as nice as just using the `--data` flag. As for initializing from a default config, this seems unwieldy - the current hard coded defaults seems reasonable for the time being